doc/Manuals/ug-syspar_x-road_v6_system_parameters.md

```diff
@@ -631,7 +631,8 @@ This chapter describes the system parameters used by the X-Road configuration pr
 
 <a id="Ref_note2"></a>[2] Default value for proxy.xroad-tls-ciphers.
 > TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384,
-> TLS_DHE_RSA_WITH_AES_256_CBC_SHA256
+> TLS_DHE_RSA_WITH_AES_256_CBC_SHA256,
```

Comment:
The version number and version history table of the document should be updated.

Reply:
updated